.gitignore

```diff
@@ -0,0 +1,7 @@
+.idea/
+.idea
+.idea/*
+laboratory.iml
```

Comment:
All files with extension "iml" must be excluded.

Reply:
i will write :  " !*.iml "  to exclude every possible .iml file